lib/bip_claims/service.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+module BipClaims
+  class Service < Common::Client::Base
+    STATSD_KEY_PREFIX = 'api.bip_claims'
+    include Common::Client::Monitoring
+
+    configuration BipClaims::Configuration
+
+    def veteran_attributes(claim)
+      case claim.form_id
+      when '21P-530'
+        ssn, full_name, bday = claim.parsed_form.values_at(
+          'veteranSocialSecurityNumber',
+          'veteranFullName',
+          'veteranDateOfBirth'
+        )
+      end
+
+      OpenStruct.new(
+        ssn: ssn,
+        first_name: full_name['first'],
+        last_name: full_name['last'],
+        birth_date: bday
+      )
+    end
+
+    def lookup_veteran_from_mvi(claim)
+      MVI::AttrService.new.find_profile(veteran_attributes(claim))&.profile
+    rescue MVI::Errors::Base
+      nil
```

Comment:
Should we do anything interesting when this happens? Update a stat counter? Log a message?

Reply:
Yes; I am not sure what sort of stats we want to capture here; it would be worth it to discuss what metrics or logging we want here. Could make sense to logging into Sentry

Reply:
Lets tick statsd counters for "mvi lookup miss" and "mvi lookup hit"

Reply:
Added